controller/router.js

```diff
@@ -95,6 +95,14 @@ module.exports = [
                         .number()
                         .integer()
                         .positive()
+                },
+                query: {
+                    fromDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD'),
+                    toDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
```

Comment:
add default values here to prevent problems in the handler.
`fromDate` should default to 1 week ago.
`toDate` should default to today.

Reference: https://github.com/hapijs/joi/blob/master/API.md#anydefaultvalue-description